scripts/obstacle_avoider.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+import rospy
+
+class ObstacleAvoider:
+
+    def get_safe_velocity(preferred_velocity):
+        print("get_safe_velocity")
```

Comment:
If you're going to use this like this, you should probably import print_function so the syntax isn't confusing